### ISSUE TITLE
Add "My Notes" the user-drop-down list

### ISSUE THREAD

**Issue by daganzdaanda** [openstreetmap/openstreetmap-website#1007]
Currently the user dropdown menu consists of "Inbox, Profile, Settings, (Home Location?), Logout"
To access my Notes, I have to go to Profile and click Notes there.
Could "Notes" be added as a direct link in the dropdown menu?

Would some of the other items from the Profile be useful in that dropdown?
I guess "Changesets" are something everyone uses, Blogs and Comments maybe not so.

**Comment by HolgerJeromin** [comment]
I would love to have changesets in this list.

**Comment by tomhughes** [comment]
It was a deliberate design decision to restrict what was on that menu and not include every link that is in the menu at the top of the user's profile page.

**Comment by daganzdaanda** [comment]
Yes, I agree if there are too many links in the dropdown, the Profile page gets kind of redundant -  unless you have a lot of "friends" and people in your area. But from a usability pov, the dropdown is just perfect for quick access to things you need often. Maybe the contents could be chosen by the user? Also, the design of the profile page comes from before the addition of the dropdown menu, am I right? So it might make sense to overhaul how these menus go together.
